airbyte-ci/connectors/pipelines/pipelines/helpers/git.py

```diff
@@ -118,6 +80,37 @@ def get_modified_files_in_pull_request(pull_request: PullRequest) -> List[str]:
     return [f.filename for f in pull_request.get_files()]
 
 
+async def checked_out_git_container(
```

Comment:
💅 Could we move this to `airbyte-ci/connectors/pipelines/pipelines/dagger/containers`

e.g. `airbyte-ci/connectors/pipelines/pipelines/dagger/containers/git.py`